package.json

```diff
@@ -97,13 +98,10 @@
     "react-turnstile": "^1.0.5",
     "react-youtube": "^9.0.3",
     "recharts": "^2.1.16",
-    "redux": "4.2.0",
     "redux-form": "8.3.8",
     "redux-logger": "^3.0.1",
     "redux-sentry-middleware": "^0.2.2",
-    "redux-thunk": "2.4.2",
     "regenerator-runtime": "^0.13.10",
-    "reselect": "4.1.7",
```

Comment:
It's included in @redux/toolkit

Reply:
Hmmm, I don't really like omiting packages just because another package includes it. IMO we should explicitly include all packages we directly depend on.